_posts/2020-10-04-newpipe-0.20.0.md

```diff
@@ -1,154 +1,176 @@
 ---
 layout: post
-title: "NewPipe 0.20.0 released: A new major version, a new unified player, a new notification experience... A lot is new this time!"
+title: "NewPipe 0.20.0 released: Major version, new unified player, new notifications. More things."
 short: "NewPipe 0.20.0 released"
 date: 2020-10-04 05:00:00 +0100
 categories: [pinned, release]
-author: Opusforlife & Poolitzer
+author: Opusforlife, Poolitzer and Allan Nordhøy
 image: newpipe
 excerpt_separator: <!-- more -->
 ---
 
-Hey there! Guess who is back? We are. Admittedly, we're quite a bit late with this release. But there is a _very_ good reason for that. Are you ready for it? Sit down, grab some popcorn and a beverage of your choice. This is going to be long.
+Hey there! Back admittedly a bit late with this release, but for _very_ good reason. Ready? Sit down, grab the virtual popcorn and some hydration.
 <!-- more -->
 
 ## A new unified player
 
-"What in the name of the coronavirus is the _unified player_?" you ask. Well, first of all, that's a very weird expression. Where did you even get that one from... - you know what? Save it. We don't want to know. You just keep on using that, we'll keep on talking about the new player.
+"If the new player is the _unified player_, was the old one un-unified?" you ask. The new player is new, and it is unified.
 
-A new player it indeed is. Up until now, NewPipe had three different players bundled in its code: the main, pop-up and background players. All three had separate code and separate playing queues. When you switched from one player to another, in the background, the old player and its queue closed itself, while a new player was created, along with a new queue, which took over then. If that sounds complicated and cumbersome, that's because it is. Moreover, if you accidentally switched using the player buttons below the thumbnail instead of the ones in the drop down menu inside the player, the stream would start from the beginning instead of carrying on from where you left, and you would lose your (possibly carefully built) queue.
+Until now, NewPipe had a main, pop-up and background -player. All three had separate code and playing queues. Switching from one player to another meant the old player and its queue closed itself, while a new player and queue was created. That sounds complicated and cumbersome, because it is. Moreover, if you accidentally switched using the player buttons below the thumbnail instead of the ones in the drop down menu inside the player, the stream would start anew instead of carrying on from where you left off. Losing your (possibly carefully built) queue was the bonus.
 
-Well, no more! The buttons in the player drop down menu have been removed and the action buttons below the thumbnail work like they're supposed to, so now you're at no risk whatsoever of losing your playing position or queue. Whew!
```

Comment:
Here it is unclear what the buttons aren't supposed to be doing. Something else happens as a result is not the same, and this is a lot of focus on what was. The point is reiterated in "this was not" and "this is" form. There is no flow there.

Reply:
It is a direct follow up to the sentence starting with "Moreover" above. Roughly: "If you did X, Y bad thing would happen. Fixed now! We have done Z so Y bad thing cannot happen".

Reply:
It reads like a https://en.wikipedia.org/wiki/Rebus where you are supposed to figure out if the giraffe lives in the garage, and what colour it is as a result of knowing other stuff.